Sources/Actions/Feed.php

```diff
@@ -1761,12 +1761,12 @@ public function getXmlProfile(): array
 				[
 					'tag' => 'last-login',
 					'attributes' => ['label' => Lang::$txt['lastLoggedIn'], 'UTC' => Time::gmstrftime('%F %T', $profile['last_login_timestamp'])],
-					'content' => Time::create('@' . $row['last_login_timestamp'], new \DateTimeZone(Config::$modSettings['default_timezone']))->format(null, false),
+					'content' => Time::create('@' . User::$me->last_login_timestamp, new \DateTimeZone(Config::$modSettings['default_timezone']))->format(null, false),
 				],
 				[
 					'tag' => 'registered',
 					'attributes' => ['label' => Lang::$txt['date_registered'], 'UTC' => Time::gmstrftime('%F %T', $profile['registered_timestamp'])],
-					'content' => Time::create('@' . $row['registered_timestamp'], new \DateTimeZone(Config::$modSettings['default_timezone']))->format(null, false),
+					'content' => Time::create('@' . User::$me->last_login_timestamp, new \DateTimeZone(Config::$modSettings['default_timezone']))->format(null, false),
```

Comment:
Maybe `User::$me->registered_timestamp`?

Reply:
Also looks like it should be $profile. Thanks!